plugins/modules/ec2_vpc_dhcp_option.py

```diff
@@ -298,116 +496,82 @@ def main():
         supports_check_mode=True
     )
 
-    params = module.params
+    vpc_id = module.params['vpc_id']
```

Comment:
Can we drop `default=None` when we replace here everything with `module.params(paraname)` or does it break some backwards compatibility?

Reply:
Do you mean like, change the arg_spec to have no defaults for these params and instead here do something like `vpc_id = module.params.get('vpc_id', None)`?